iso creation for latest build and cleaning the scripts.

naveenmh opened the pull request.

naveenmh committed:
iso creation for latest build and cleaning the scripts.
- updated looping disks while selecting the os drive during install
- partition scheme change for creating /tmp /var for fixed size and removed /home
- partition scheme change now will have remaining unused space for creating /slog and /l2arc
- removed execute permission on tgtd.service and urbackup-server.service
- removed 'TasksMax' parameter in urbackup-server.service as its not supported for kernel v.216
- now we have dell specific dependencies to be installed only on dell platform
- updated ks for latest changes on partitioning and removed git strings from the ks file.

raamsri commented on install/iso_creation/cd_or_vm/ks.cfg:
```diff
@@ -69,22 +69,62 @@ else
 
+
+### Running a for loop on Disks available among one will be selected by user/admin for OS to Install ###
+<<<<<<< 16f4fc60b4242c94fdb1501886ca305d0ab90a72
```
You need to properly resolve the git merge conflicts. I'm quite sure this script will not execute successfully with all the git strings in it.

naveenmh replied:
Surprise...I amended the last commits to one.

raamsri commented on install/iso_creation/cd_or_vm/ks.cfg:
```diff
@@ -69,22 +69,62 @@ else
+
+### Running a for loop on Disks available among one will be selected by user/admin for OS to Install ###
+<<<<<<< 16f4fc60b4242c94fdb1501886ca305d0ab90a72
+=======
```
Resolve.

raamsri commented on install/iso_creation/cd_or_vm/ks.cfg:
```diff
@@ -69,22 +69,62 @@ else
+echo "Disk Name         Size"
+for DEV in ${DISKS[@]} ; do
+  if [ -b "/dev/$DEV" ] ; then
+<<<<<<< 16f4fc60b4242c94fdb1501886ca305d0ab90a72
```
Resolve.

raamsri commented on install/iso_creation/cd_or_vm/ks.cfg:
```diff
@@ -69,22 +69,62 @@ else
+    echo "   "$DEV"             "$GB"GB"
+   fi
+done
+<<<<<<< 16f4fc60b4242c94fdb1501886ca305d0ab90a72
```
Resolve.

raamsri commented on install/iso_creation/cd_or_vm/ks.cfg:
```diff
@@ -69,22 +69,62 @@ else
+                echo "   "$device"              "$GB"GB"
+            fi
+    fi
+<<<<<<< 16f4fc60b4242c94fdb1501886ca305d0ab90a72
```
Resolve.

raamsri reviewed (changes requested):
Please amend this commit to fix the code instead of adding a new one on top of this commit since this has erroneous code; let's try to have only valid, clean and necessary commits in the repo.

naveen-fractlio commented:
Updated the changed requested.

naveenmh committed:
integralstor rpm creation for new code
- added code/software update by git pull - by this allowing user to pull code before running rpm script
- cleanup the code and now allows to have a flexible working directory for rpm creation
- no manual process in creating or installing when we have all the dependencies installed.

naveenmh committed:
Added post operation script for ks file

- Added AFPD and logwatch functionality.

naveenmh committed:
Merge branch 'master' into master

naveenmh committed:
Update integralstor_ks_post.sh
